src/directives.js

```diff
@@ -0,0 +1,33 @@
+(function(){
+	let app = angular.module('invertedIndexApp');
+
+	/*
```

Comment:
Your doc string comment should start with a forward slash followed by double asterisks
like this  
`/**` 
not like this
`/*`
And make sure they are properly aligned.